app/uk/gov/hmrc/helptosavestub/controllers/NSIController.scala

```diff
@@ -107,6 +112,75 @@ object NSIController extends BaseController with Logging {
 
   private val ninoStatusRegex = """AS(\d{3}).*""".r
 
+  def getAccount(correlationId: Option[String], nino: Option[String], version: Option[String], systemId: Option[String]): Action[AnyContent] = Action { implicit request ⇒
+    validateParams(correlationId, nino, version, systemId)
+      .fold(
+        errors ⇒ {
+          logger.warn("[Handle Account Query] invalid params")
+          errors match {
+            case error :: Nil ⇒ BadRequest(Json.toJson(error))
+            case _            ⇒ BadRequest
```

Comment:
I'm not sure omitting the body altogether is the right thing to do when there are multiple errors. However the ICD doesn't specify what the behaviour should be in this case so I've asked Rob. I won't wait for his reply before merging this, we can update the stub to match the behaviour he specifies in a new PR.

Reply:
ok